Mathlib/Algebra/GroupWithZero/Power.lean

```diff
@@ -197,10 +185,5 @@ theorem div_sq_cancel (a b : G₀) : a ^ 2 * b / a = a * b := by
 
 end
 
-/-- If a monoid homomorphism `f` between two `GroupWithZero`s maps `0` to `0`, then it maps `x^n`,
-`n : ℤ`, to `(f x)^n`. -/
-@[simp]
-theorem map_zpow₀ {F G₀ G₀' : Type*} [GroupWithZero G₀] [GroupWithZero G₀']
-    [MonoidWithZeroHomClass F G₀ G₀'] (f : F) (x : G₀) (n : ℤ) : f (x ^ n) = f x ^ n :=
-  map_zpow' f (map_inv₀ f) x n
-#align map_zpow₀ map_zpow₀
```

Comment:
Why did this lemma move? The new home for it looks a bit weird to me.

Reply:
It moved upstream (to the highest place it could be defined) so that `Mathlib/Data/Rat/Cast/CharZero.lean` does not need to import this file.

If you make a follow up PR moving it somewhere else without increasing the imports of `Data.Rat.Cast.CharZero` I'll merge it asap.